### PR TITLE
Implement CircleCi configuration

### PR DESCRIPTION
**Pull request by simonovic86**
- Implement CircleCi configuration

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,117 @@
+version: 2.1
+
+orbs:
+  aws-ecr: circleci/aws-ecr@6.8.1
+  aws-cli: circleci/aws-cli@1.0.0
+  slack: circleci/slack@3.4.2
```
Comment (oed) [inline .circleci/config.yml:6]: Why is this here?

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+            - push-dev-image
+          filters:
+            branches:
+              only: develop
+      - run-smoke-test:
+          requires:
+            - deploy-dev
+          filters:
+            branches:
+              only: develop
```
Comment (oed) [inline .circleci/config.yml:111]: Do we need to run the smoke tests here?

Comment (simonovic86) [reply .circleci/config.yml:111]: no, I'll remove that

File: .circleci/config.yml (hunk 1 above)
Comment (simonovic86) [reply .circleci/config.yml:6]: `slack` doesn't need to be here, I'll remove it :)
